lib/logger/lib/logger/translator.ex

```diff
@@ -307,17 +328,26 @@ defmodule Logger.Translator do
       is_integer(args) ->
         format_undef(mod, fun, args, undef, stacktrace)
       true ->
-        Exception.format_exit(reason)
+        format_stop_banner(undef)
     end
   end
 
-  defp format_otp_exit(reason) do
-    Exception.format_exit(reason)
+  defp format_stop_banner(reason, stacktrace) do
+    if Exception.exception?(reason) do
+        Exception.format_banner(:error, reason, stacktrace)
+    else
+      case Exception.normalize(:error, reason, stacktrace) do
+        %ErlangError{} ->
+          format_stop_banner(reason)
```

Comment:
Given all those points, then think the current code is fine, we try to guess a bit, otherwise we don't. There is a chance of false positives but they are tiny (we don't normalize many).

Also, given we may have our own GenServer implementation by 1.2, should we love some code comments here explaining this could be cleaned up once we do?